cypress/support/cypress-commands/lifecycle.js

```diff
@@ -506,6 +506,7 @@ Cypress.Commands.add('fetchLifecycleHistory', (appId) => {
  * @example
  * cy.setAppIdFromAppType('3rd party app')
  * cy.setAppIdFromAppType('1st party app')
+ * cy.setAppIdFromAppType('player')
  */
```

Comment:
not sure again why player is being used in FCS. Give a dummy name here

Reply:
done